src/util/virutil.c

```diff
@@ -277,6 +277,67 @@ virParseVersionString(const char *str, unsigned long *version,
     return 0;
 }
 
+int
+virCHVersionString(const char *str, unsigned long *version,
+                      bool allowMissing)
+{
+    int ret = virParseVersionString(str, &version, true);
+    if ( ret < 0){
+        unsigned int major, minor = 0, micro = 0;
+
+        if (strncmp(str, "cloud-hypervisor v", 18) == 0) {
```

Comment:
The input `str` here already has this `cloud-hypervisor v` stripped out at the upper scope.

Thanks